src/plugins/intel_cpu/src/nodes/mkldnn_roi_pooling_node.cpp

```diff
@@ -306,6 +307,62 @@ struct jit_uni_roi_pooling_kernel_f32 : public jit_uni_roi_pooling_kernel, publi
     }
 };
 
+namespace {
+struct RoiPoolingKey {
+    jit_roi_pooling_params refParams;
+
+    size_t hash() const;
+    bool operator==(const RoiPoolingKey& rhs) const;
+};
+
+size_t RoiPoolingKey::hash() const {
+    using namespace dnnl::impl;
+    using namespace dnnl::impl::primitive_hashing;
+
+    size_t seed = 0;
+
+    seed = hash_combine(seed, refParams.mb);
+    seed = hash_combine(seed, refParams.c);
+    seed = hash_combine(seed, refParams.nb_c);
+    seed = hash_combine(seed, refParams.ih);
+    seed = hash_combine(seed, refParams.iw);
+    seed = hash_combine(seed, refParams.oh);
+    seed = hash_combine(seed, refParams.ow);
+    seed = hash_combine(seed, refParams.alg);
+    seed = hash_combine(seed, refParams.src_prc.getPrecVal());
+    seed = hash_combine(seed, refParams.dst_prc.getPrecVal());
+    seed = hash_combine(seed, refParams.spatial_scale);
+    seed = hash_combine(seed, refParams.pooled_h);
+    seed = hash_combine(seed, refParams.pooled_w);
+
+    return seed;
+}
+
+bool RoiPoolingKey::operator==(const RoiPoolingKey &rhs) const {
+    return refParams == rhs.refParams;
+}
+} // namespace
+
+bool MKLDNNPlugin::jit_roi_pooling_params::operator==(const MKLDNNPlugin::jit_roi_pooling_params &rhs) const noexcept {
+    return mb == rhs.mb &&
+           c == rhs.c &&
+           ih == rhs.ih &&
+           iw == rhs.iw &&
+           oh == rhs.oh &&
+           ow == rhs.ow &&
+           c_block == rhs.c_block &&
+           nb_c == rhs.nb_c &&
+           nb_c_blocking == rhs.nb_c_blocking &&
+           spatial_scale == rhs.spatial_scale &&
+           pooled_h == rhs.pooled_h &&
+           pooled_w == rhs.pooled_w &&
+           src_prc == rhs.src_prc &&
+           dst_prc == rhs.dst_prc &&
+           src_data_size == rhs.src_data_size &&
+           dst_data_size == rhs.dst_data_size &&
```

Comment:
I guess `src_data_size` and `dst_data_size` are excess in `jit_roi_pooling_params` struct. We can just call src_prc.size() inside executor. May I ask you to refactor this?

Reply:
Sure, I removed `src_data_size` and `dst_data_size` from `jit_roi_pooling_params` struct, and I also removed `src_data_size` and `dst_data_size` from private members because seems they are not been used anywhere.